css-custom-properties/index.html

```diff
@@ -0,0 +1,200 @@
+    <div class="cell-actions">
+      <button class="picker-button">
+        Use this color
+        <div class="ripple"></div>
+      </button>
+    </div>
+  </div>
+  <div class="cell" style="--primary-color: #E91E63; --primary-color-text: #FFF;">
+    <header class="cell-header">
+      <div class="cell-title">
+        Pink
+      </div>
+    </header>
+    <main class="cell-content">
+      The colors on the cards are not affected because they're individually
+      defined at the card level, and thus prioritised according to standard CSS
+      rules.
+    </main>
+    <div class="cell-actions">
+      <button class="picker-button">
+        Use this color
+        <div class="ripple"></div>
+      </button>
+    </div>
+  </div>
+  <div class="cell" style="--primary-color: #9C27B0; --primary-color-text: #FFF;">
+    <header class="cell-header">
+      <div class="cell-title">
+        Purple
+      </div>
+    </header>
+    <main class="cell-content">
+      Use the controls above to adjust some properties that affect the entire
+      page.
+    </main>
+    <div class="cell-actions">
+      <button class="picker-button">
+        Use this color
+        <div class="ripple"></div>
+      </button>
+    </div>
+  </div>
+  <div class="cell" style="--primary-color: #00BCD4; --primary-color-text: #424242;">
+    <header class="cell-header">
+      <div class="cell-title">
+        Cyan
+      </div>
+    </header>
+    <main class="cell-content">
+      Every size on the page is calculated as a multiple of the spacing unit,
+      so setting the property to a different value resizes all elements.
+    </main>
+    <div class="cell-actions">
+      <button class="picker-button">
+        Use this color
+        <div class="ripple"></div>
+      </button>
+    </div>
+  </div>
+  <div class="cell" style="--primary-color: #009688; --primary-color-text: #FFF;">
+    <header class="cell-header">
+      <div class="cell-title">
+        Teal
+      </div>
+    </header>
+    <main class="cell-content">
+      Changing the number of columns is done by recalculating the relative size
+      of cells on the grid.
+    </main>
+    <div class="cell-actions">
+      <button class="picker-button">
+        Use this color
+        <div class="ripple"></div>
+      </button>
+    </div>
+  </div>
+  <div class="cell" style="--primary-color: #4CAF50; --primary-color-text: #424242;">
+    <header class="cell-header">
+      <div class="cell-title">
+        Green
+      </div>
+    </header>
+    <main class="cell-content">
+      Margins adjusts both the spacing around the grid and the gutters inside of
+      it.
+    </main>
+    <div class="cell-actions">
+      <button class="picker-button">
+        Use this color
+        <div class="ripple"></div>
+      </button>
+    </div>
+  </div>
+</div>
+{% endcapture %}
+{% include output_helper.html initial_output_content=initial_output_content %}
+
+<h3>CSS Snippet</h3>
+
+{% highlight css %}
```

Comment:
Instead of including an extra copy of the CSS inside of the `{% highlight css %}` block, you can use the `css_snippet.html` helper to both display the highlighted CSS inline, as well as include it in the page via a `<style>` tag.

https://github.com/GoogleChrome/samples/blob/gh-pages/css-motion-path/index.html#L59 is an example.

Given that the CSS is so important to this example, I'd recommend doing away with the external `index.css` file entirely, and put all of your CSS inline here using `css_snippet.html`.